matelau/goods_crawler
# Create web ui

## Changes 
1. Added create-react-app in the web directory
## How to test
1. cd into web directory
2. npm install
3. run yarn start
4. verify app start on local host
